docs/fastddscli/cli/cli.rst

```diff
@@ -0,0 +1,139 @@
+.. include:: ../../03-exports/roles.include
+
+.. _ffastddscli_cli:
+
+CLI
+===
+
+The *Fast DDS* command line interface provides a set commands and sub-commands to perform, *Fast DDS*
+related, maintenance and configuration tasks.
+
+An executable file for Linux and Windows that runs the *Fast DDS CLI* application is
+available in the ``tools`` folder.
+If the ``tools/fastdds`` folder path is added to the ``PATH``, or by sourcing the `<path/to/fastdds>/install/setup.bash`
+environment variable, *Fast DDS CLI* can be executed running the following commands:
+
+- Linux:
+
+    .. code-block:: bash
+
+        $ fastdds <command> [<command-args>]
+
+-  Windows:
+
+    .. code-block:: bash
+
+        > fastdds.bat <command> [<command-args>]
+
++--------------------------+-------------------------------------------------------------------------------------------+
+| Verbs                    | Description                                                                               |
++==========================+===========================================================================================+
+| ``discovery``            | Launches a server for :ref:`Server-Client Discovery<discovery_server>`.                   |
++--------------------------+-------------------------------------------------------------------------------------------+
+| ``shm``                  | Allows manual cleaning of garbage files that may be generated by                          |
+|                          | :ref:`transport_sharedMemory_sharedMemory`                                                |
++--------------------------+-------------------------------------------------------------------------------------------+
+
+.. _cli_discovery:
+
+discovery
+---------
+
+Launches a server for :ref:`Server-Client Discovery<discovery_server>`. This server will manage the messages of the
+clients which are pointed to its IP address.
+Clients must be aware of how to reach the server by specifying an IP address and a transport protocol like UDP
+or TCP. Servers don’t need any beforehand knowledge of their clients but require a listening IP address, where they
+may be reached, to be specified.
```

Comment:
Refer to Discovery section https://github.com/eProsima/Fast-RTPS-docs/pull/184#discussion_r462940374